File: cop_mobile/cop_mobile/cop_mobile/Model/DepartmentData.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace cop_mobile.Model
{
    public class DepartmentData
    {
        private List<Department> depts;
        private CustomPage<Member> members;

        public DepartmentData(List<Department> depts, CustomPage<Member> members)
        {
            Depts = depts;
            Members = members;
        }

        public DepartmentData()
        {

        }

        public List<Department> Depts { get => depts; set => depts = value; }
        public CustomPage<Member> Members { get => members; set => members = value; }
    }
}

File: web-server/src/main/java/com/churchofphilippi/webserver/model/customModels/DepartmentData.java
package com.churchofphilippi.webserver.model.customModels;

import com.churchofphilippi.webserver.model.Dept;
import com.churchofphilippi.webserver.model.Member;
import com.churchofphilippi.webserver.model.pagination.CustomPage;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.data.domain.Page;

import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class DepartmentData {
    private List<Dept> depts;
    private CustomPage<Member> members;
}
